frontend/public/images/quotes/antonopoulos.svg

```diff
@@ -1 +1,4 @@
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="667" height="800" fill="none">
+  <path fill="gray"
```

Comment:
I think we should also use `fill=currentColor"` so we can specify it. Gray seems to not be so clear in light vs dark mode.

Reply:
We should make it into an icon if we want thi currentColor to work, because we are currently passing this svg as an src to img tag , should I make this white instead of gray? It would look like this then:

<img width="100%" alt="Screenshot 2024-09-25 at 11 54 16 AM" src="https://github.com/user-attachments/assets/8478f1c6-64f9-42b7-9e4a-10f9238a207a">

Compared to what we have right now:
<img width="100%" alt="Screenshot 2024-09-25 at 11 56 59 AM" src="https://github.com/user-attachments/assets/94097f67-b72b-4828-8a54-c98174ddae94">

Reply:
@im-adithya this won't work on dark mode, right?

Actually I was wrong, if `gray` is completely half way between black and white it should be ok. We might just need to make the opacity stronger in light mode vs dark mode, since it seems less visible there.

> HTML / CSS defined gray at the midpoint of the 8-bit gray scale (128,128,128).

Reply:
Ah yes it wouldn't, but actually the current one as is looks good both on Light Mode and Dark Mode

<table>
<tr>
<td>
<h3>Light Mode</h3>
</td>
<td>
<h3>Dark Mode</h3>
</td>
</tr>

<tr>
<td>
<img width="100%" alt="Screenshot 2024-09-25 at 12 58 06 PM" src="https://github.com/user-attachments/assets/7149ba8f-92e7-4087-aa66-6f9e52d6e8e9">
</td>

<td>
<img width="100%" alt="Screenshot 2024-09-25 at 12 58 43 PM" src="https://github.com/user-attachments/assets/f7104e1d-eeed-4869-978a-2357a5936eac">
</td>

</tr>
</table>

Reply:
@im-adithya the light and dark mode are back to front, but the one with the black background is very hard to see the silhoutte. Can we use a different opacity on that one? e.g. `opacity-40 dark:opacity-20`

Reply:
Done! Looks better now!


<table>
<tr>
<td>
<h3>Light Mode</h3>
</td>
<td>
<h3>Dark Mode</h3>
</td>
</tr>

<tr>
<td>
<img width="100%" alt="Screenshot 2024-09-26 at 1 45 14 PM" src="https://github.com/user-attachments/assets/0f7955ba-f507-4ea9-acd7-8dd9c928dbc1">
</td>

<td>
<img width="100%" alt="Screenshot 2024-09-26 at 1 45 30 PM" src="https://github.com/user-attachments/assets/518a067a-6a9a-46b4-87af-3e79dad1d63d">
</td>

</tr>
</table>